staging/src/k8s.io/apiserver/pkg/endpoints/filterlatency/filterlatency.go

```diff
@@ -24,13 +24,17 @@ import (
 	utilclock "k8s.io/apimachinery/pkg/util/clock"
 	"k8s.io/apiserver/pkg/endpoints/metrics"
 	apirequest "k8s.io/apiserver/pkg/endpoints/request"
+	"k8s.io/apiserver/pkg/server/httplog"
+	"k8s.io/klog/v2"
 )
 
 type requestFilterRecordKeyType int
 
 // requestFilterRecordKey is the context key for a request filter record struct.
 const requestFilterRecordKey requestFilterRecordKeyType = iota
 
+const minFilterLatencyToLog = time.Duration(100) * time.Millisecond
```

Comment:
Nit: 100 * time.Millisecond, there is no need for casting here and in fact it's quite uncommon to do cast. (grep for "*time.Millisecond" in codebase).